### ISSUE TITLE
What is the recent V2 model for?

### ISSUE THREAD

**Issue by johnpaulbin** [sberbank-ai/ru-dalle#50]
As seen inside https://huggingface.co/sberbank-ai/rudalle-Malevich/tree/main

What does the new pytorch model (v2) bring?

**Comment by blue-fish** [comment]
It's for compatibility with #47. The names of the mask parameters changed in the model dict.

`row_mask` --> `transformer.row_mask`
`col_mask` --> `transformer.col_mask`
`conv_mask` --> `transformer.row_mask`

You can verify this for yourself with this code.
https://gist.github.com/blue-fish/b07abadab81f3fda365adc87f2b500e0

Output:
```
Key: row_mask in model_v1, but not model_v2
Key: col_mask in model_v1, but not model_v2
Key: conv_mask in model_v1, but not model_v2
Key: transformer.row_mask in model_v2, but not model_v1
Key: transformer.col_mask in model_v2, but not model_v1
Key: transformer.conv_mask in model_v2, but not model_v1
Checking: model_v1["row_mask"] vs. model_v2["transformer.row_mask"]
... exact match
Checking: model_v1["col_mask"] vs. model_v2["transformer.col_mask"]
... exact match
Checking: model_v1["conv_mask"] vs. model_v2["transformer.conv_mask"]
... exact match
```

**Comment by shonenkov** [comment]
@johnpaulbin  @blue-fish yes, it is some refactoring only ;) without new pretrained models
